### DISCUSSION TITLE
Service Cloud Connector - Sync status

### DISCUSSION THREAD

**Discussion by shayunBBD** [SalesforceCommerceCloud/sfcc-ci#422] [Team Posts]
Hello,

When I create a new cutomer in CC, the sync status is by default -None-.
I saw that it is sent to SC just if I switch the status to created.
Why? If I import 200 customers, should I change every one to synch it with SC?
There is a way maybe to set created as default value?

Thank you

**Comment by alicesong11** [comment]
Hello,

How did you create your customer? Directly in BM(business management) of CC or via the website? Try to create 1 customer via website then check if the status of customer becomes "created" automatically.

Regards

**Comment by shayunBBD** [comment]
I created via the storefront but it's still with -none-

**Comment by alicesong11** [comment]
Ok, juste try to make a purchase on storefront with the account you created then check if status changes?

**Comment by shayunBBD** [comment]
We don't have a products in our sb. I'll check that, but why should I make a purchase to synch the customer? I can't synch automatically a new customer? Maybe set created as default value? It's possible? how?

**Comment by alicesong11** [comment]
Go to the storefront and click on "nouveauté", you will find available products. Logically, when a new customer created, the customer service needs to know the customer when there is a problem with his or her order. I'm not sure about that, but try to make a purchase then check the status please

**Comment by pravngaur** [comment]
The customer status is created for me always and it gets synched in SC as well, do not think making purchase should have any effect. Could you please debug & check your cartridge path and see if the plugin-connector cartridge(if you are on SFRA) is in effect because all the relevant code is there.

What I suspect is that your base cartridge is taking precedence, therefore, connector specific things are not happening. Check please

**Comment by shayunBBD** [comment]
doesn't work :( still none

**Comment by pravngaur** [comment]
Hi Shayn, please debug and see if the plugin cartridge Account.js controller is in effect or not. Because if that code is executing, then it should be updated. 

Check this and also check logs for any error.

**Comment by shayunBBD** [comment]
@pravngaur Thank you. I added breakpoint in accountCreated and handleExport functions. When I create a new customer via the storefront the code is not executing. Do you have a solution?

**Comment by shayunBBD** [comment]
What should I do to execute this code? just create a new customer from my storefront?

**Comment by shayunBBD** [comment]
When I try to create a new customer I receive this error:
ERROR PipelineCallServlet|1089026932|Sites-...-Site|Account-StartRegister|PipelineCall|obn84trqu2 custom.CRM []  SoapRequest.FetchRequest

**Comment by shayunBBD** [comment]
@pravngaur can you please help me?
